azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -125,7 +125,7 @@
       "azkaban.webapp.servlet.LoginAbstractAzkabanServlet";
   private static final Logger logger = Logger.getLogger(AzkabanWebServer.class);
   private static final int MAX_FORM_CONTENT_SIZE = 10 * 1024 * 1024;
-  private static final String DEFAULT_TIMEZONE_ID = "default.timezone.id";
+  static final String DEFAULT_TIMEZONE_ID = "default.timezone.id";
```

Comment:
What is the need to remove private qualifier?

Reply:
It is being set in AzkabanWebServerTest to make test cases system independent.

Reply:
I am against the philosophy to modify product behavior to make test(s) work. It should be the other way around.

Reply:
Very well, I have copied the string literal and re-marked the field as private.